modules/kubernetes-addons/istio/main.tf

```diff
@@ -0,0 +1,138 @@
+resource "kubernetes_namespace" "istio_system" {
+  metadata {
+    name = "istio-system"
+    labels = {
+      istio-injection = "enabled"
+    }
+  }
+}
+
+module "istio_base" {
+  source = "../helm-addon"
+  count  = var.install_istio_base ? 1 : 0
+
+  helm_config = merge(
+    {
+      name            = "istio-base"
+      chart           = "base"
+      repository      = "https://istio-release.storage.googleapis.com/charts"
+      version         = var.istio_version
+      namespace       = kubernetes_namespace.istio_system.metadata.0.name
+      timeout         = 120
+      cleanup_on_fail = var.cleanup_on_fail
+      force_update    = var.force_update
+      values = [
+        yamlencode(var.istio_base_settings)
+      ]
+      description = "Helm chart for deploying Istio cluster resources and CRDs"
+    },
+    var.helm_config
+  )
+  manage_via_gitops = var.manage_via_gitops
+  addon_context     = var.addon_context
+
+  depends_on = [kubernetes_namespace.istio_system]
+}
+
+module "istio_cni" {
+  source = "../helm-addon"
+  count  = var.install_istio_cni ? 1 : 0
+
+  helm_config = merge(
+    {
+      name            = "istio-cni"
+      chart           = "cni"
+      repository      = "https://istio-release.storage.googleapis.com/charts"
+      version         = var.istio_version
+      namespace       = kubernetes_namespace.istio_system.metadata.0.name
+      timeout         = 120
+      cleanup_on_fail = var.cleanup_on_fail
+      force_update    = var.force_update
+      values = [
```

Comment:
is this value required to function correctly? if not, don't set it as a default and let users pass in their own values